lab-brianna/server.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const bodyParser = require('./lib/body-paser');
```

Comment:
Error: can't find module body-"paser".  can't get to internals of the file if the pathname isn't spelled correctly. it happens!